react/features/feedback/actions.js

```diff
@@ -125,16 +154,17 @@ export function submitFeedback(
         score: number,
         message: string,
         conference: Object) {
-    return (dispatch: Dispatch<any>) => conference.sendFeedback(score, message)
-        .then(
-            () => dispatch({ type: SUBMIT_FEEDBACK_SUCCESS }),
-            error => {
-                dispatch({
-                    type: SUBMIT_FEEDBACK_ERROR,
-                    error
-                });
-
-                return Promise.reject(error);
-            }
-        );
+    return (dispatch: Dispatch<any>) =>
+        conference.sendFeedback(score, message)
+        .then(() => dispatch({ type: SUBMIT_FEEDBACK_SUCCESS }))
+        .then(() => dispatch(sendJaasFeedbackMetadata(conference, { score,
```

Comment:
Haven't we dispatched `sendJaasFeedbackMetadata` already trough  `conference.sendFeedback`?

Reply:
As stated in my previous comment, think the conference.sendFeedback is not used anywhere. In the Feedback actions we call the .sendFeedback from conference.room ( JItsiConference instance) which calls the low level lib-jitsi-meet function.